css-two-column-layout/styles.css

```diff
@@ -0,0 +1,66 @@
+section {
+  padding-top: 2px;
+  text-align: center;
+  padding-bottom: 80px;
+}
+
+label {
+  color: rgb(90 88 88);
+  font-family: Karla;
+}
+
+h1 {
+  padding-top: 25px;
+  text-align: center;
+  font-size: 43px;
+  font-family: Karla;
+  font-style: normal;
+  font-weight: 700;
+}
+
+p {
+  font-size: 15px;
+  font-family: Karla;
+  font-style: normal;
+  font-weight: 400;
+  color: rgb(90 88 88);
+  line-height: 1.5;
+}
+
+h2,
+h3,
+h4,
+h5 {
+  font-size: 15px;
+  padding-top: 20px;
+  font-family: Karla;
+  font-style: normal;
+  font-weight: 700;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  max-width: 1500px;
+  margin-left: 100px;
+}
+
+.row {
+  display: flex;
+  flex-basis: calc(100% / 2);
```

Comment:
the row class should take up all of the content. instead of setting this to 50% width, wrap the content as noted above in the comment to get the two column layout.